docs/descriptors/xb1sx.md

```diff
@@ -0,0 +1,167 @@
+0xA1, 0x00,        //   Collection (Physical)
+0x09, 0x33,        //     Usage (Rx)
+0x09, 0x34,        //     Usage (Ry)
+0x15, 0x00,        //     Logical Minimum (0)
+0x27, 0xFF, 0xFF, 0x00, 0x00,  //     Logical Maximum (65534)
+0x95, 0x02,        //     Report Count (2)
+0x75, 0x10,        //     Report Size (16)
+0x81, 0x02,        //     Input (Data,Var,Abs,No Wrap,Linear,Preferred State,No Null Position)
+0xC0,              //   End Collection
+0x05, 0x01,        //   Usage Page (Generic Desktop Ctrls)
+0x09, 0x32,        //   Usage (Z)
+0x15, 0x00,        //   Logical Minimum (0)
+0x26, 0xFF, 0x03,  //   Logical Maximum (1023)
+0x95, 0x01,        //   Report Count (1)
+0x75, 0x0A,        //   Report Size (10)
+0x81, 0x02,        //   Input (Data,Var,Abs,No Wrap,Linear,Preferred State,No Null Position)
+0x15, 0x00,        //   Logical Minimum (0)
+0x25, 0x00,        //   Logical Maximum (0)
+0x75, 0x06,        //   Report Size (6)
+0x95, 0x01,        //   Report Count (1)
+0x81, 0x03,        //   Input (Const,Var,Abs,No Wrap,Linear,Preferred State,No Null Position)
+0x05, 0x01,        //   Usage Page (Generic Desktop Ctrls)
+0x09, 0x35,        //   Usage (Rz)
+0x15, 0x00,        //   Logical Minimum (0)
+0x26, 0xFF, 0x03,  //   Logical Maximum (1023)
+0x95, 0x01,        //   Report Count (1)
+0x75, 0x0A,        //   Report Size (10)
+0x81, 0x02,        //   Input (Data,Var,Abs,No Wrap,Linear,Preferred State,No Null Position)
+0x15, 0x00,        //   Logical Minimum (0)
+0x25, 0x00,        //   Logical Maximum (0)
+0x75, 0x06,        //   Report Size (6)
+0x95, 0x01,        //   Report Count (1)
+0x81, 0x03,        //   Input (Const,Var,Abs,No Wrap,Linear,Preferred State,No Null Position)
+0x05, 0x01,        //   Usage Page (Generic Desktop Ctrls)
+0x09, 0x39,        //   Usage (Hat switch)
+0x15, 0x01,        //   Logical Minimum (1)
+0x25, 0x08,        //   Logical Maximum (8)
+0x35, 0x00,        //   Physical Minimum (0)
+0x46, 0x3B, 0x01,  //   Physical Maximum (315)
+0x66, 0x14, 0x00,  //   Unit (System: English Rotation, Length: Centimeter)
+0x75, 0x04,        //   Report Size (4)
+0x95, 0x01,        //   Report Count (1)
+0x81, 0x42,        //   Input (Data,Var,Abs,No Wrap,Linear,Preferred State,Null State)
+0x75, 0x04,        //   Report Size (4)
+0x95, 0x01,        //   Report Count (1)
+0x15, 0x00,        //   Logical Minimum (0)
+0x25, 0x00,        //   Logical Maximum (0)
+0x35, 0x00,        //   Physical Minimum (0)
+0x45, 0x00,        //   Physical Maximum (0)
+0x65, 0x00,        //   Unit (None)
+0x81, 0x03,        //   Input (Const,Var,Abs,No Wrap,Linear,Preferred State,No Null Position)
+0x05, 0x09,        //   Usage Page (Button)
+0x19, 0x01,        //   Usage Minimum (0x01)
+0x29, 0x0B,        //   Usage Maximum (0x0B)
+0x15, 0x00,        //   Logical Minimum (0)
+0x25, 0x01,        //   Logical Maximum (1)
+0x75, 0x01,        //   Report Size (1)
+0x95, 0x0B,        //   Report Count (11)
```

Comment:
It reports 11 buttons here. This seems much more sane than the XBE2 report which says it has 15 buttons. It thus matches with the Windows mode of the older controllers, and the original non-wireless controllers. But it seems to put the share button in an unexpected position. You probably need to fix that to report 12 buttons instead, so we can patch the raw events to reports the 10 standard bits correctly, plus bit 11 for the logo button, and bit 12 for the share button.